syntax = "proto3";
package transfer;

message ResponseEmptyMessage
{
}
message TransferMessage
{
  string message_name = 1;
  bytes message_content = 2;
}

message infoString{
  string msg = 1;
}